lib/middleware/backbone_request_id.js

```diff
@@ -0,0 +1,11 @@
+import Backbone from 'backbone'
+import { wrap } from 'underscore'
+
+module.exports = (req, res, next) => {
+  Backbone.sync = wrap(Backbone.sync, (sync, method, model, options = {}) => {
```

Comment:
Also let me know if there is a better way of doing this-- basically I want to be able to set the `X-Request-Id` header on every call that gets made through Backbone.